.github/workflows/copy-solution-scripts.yml

```
@@ -0,0 +1,84 @@
+name: Copy solution scripts to rrp-workshop-exercises
+# Copy instructor_notes/solutions/*sh to rrp-workshop-exercises repo
+
+# This workflow will run when there are changes to solution scripts in THIS repo
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "instructor_notes/solutions/*sh"
+
+jobs:
+  file-script-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Load 1Password secrets
+        uses: 1password/load-secrets-action@v1
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.TRAINING_OP_SERVICE_ACCOUNT_TOKEN }}
+          ACTION_MONITORING_SLACK: ${{ secrets.OP_ACTION_MONITORING_SLACK }}
+
+      - name: Checkout code from rrp-workshop-exercises
+        uses: actions/checkout@v4
+        with:
+          repository: AlexsLemonade/rrp-workshop-exercises
+          token: ${{ env.DOCS_BOT_GITHUB_TOKEN }}
+
+      - name: Configure git
+        run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
+
+      - name: Get solution scripts from reproducible-research
+        shell: bash
+        run: |
+          # the url from this repo where files will be downloaded from
+          base_url=https://raw.githubusercontent.com/${GITHUB_REPOSITORY}/${GITHUB_REF#refs/heads/}
+
+          script_paths=$(ls ${instructor_notes/solutions}/*.sh)
```

I assume this is not a variable?
```suggestion
          script_paths=$(ls instructor_notes/solutions/*.sh)
```